Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10
```

Comment:
seems that this `Dockefile` is used for e2e testing only. Please consider to have a separate directory `e2e` which will be a root for everything that is related to e2e testing. So this file will located there as well as the directories `parity`, `ui` (were `ui-e2e`) and `oracle` (were `oracle-e2e`).

Reply:
Seems like wrong direction to me because:
- we need this Dockerfile in root dir because otherwise we wouldn't be able to access `yarn.lock` and other sub-dirs that are subject to e2e testing
![Screenshot 2019-05-22 at 10 24 57](https://user-images.githubusercontent.com/12039224/58159032-f08b6680-7c7b-11e9-9e78-fb6e15b45ec7.png)
Not sure if we can or want to fight against docker in this regard.
- goes against flat repository structure
- not sure how `yarn workspaces` would work with `package.json` located two directory levels down (instead of just one down)

Reply:
Ok. Don't you mind to rename the docker file to `Dockerfile.e2e` in this case?